### PR TITLE
handle download

### PR DESCRIPTION
**Pull request by nguyenvanhieu97**
https://edu-redmine.framgia.vn/issues/3517
https://edu-redmine.framgia.vn/issues/3518

### CODE REVIEW

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/java/com/cris/nvh/framgiaproject/mediaplayer/MediaPlayerManager.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public static MediaPlayerManager getsInstance(Context context) {
 	public void create(int index) {
 		mCurrentIndex = index;
 		Track track = mTracks.get(mCurrentIndex);
-		mListener.onStartLoading();
+//		mListener.onStartLoading();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/mediaplayer/MediaPlayerManager.java]: remove

File: app/src/main/java/com/cris/nvh/framgiaproject/service/DownloadService.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+				String newUrl = connection.getHeaderField(LOACATION_HEADER);
+				connection = (HttpURLConnection) new URL(newUrl).openConnection();
+			}
+			InputStream input = new BufferedInputStream(connection.getInputStream());
+			downloadFile(input, track.getTitle(), connection.getContentLength());
+			input.close();
+			connection.disconnect();
+		} catch (IOException e) {
+			sendRequest(EXTRA_ERROR,
+				e.getMessage(), ERROR);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/service/DownloadService.java]: enter tùy tiện

**Comment by NguyenMinhTuanB** [comment]
commented

**Pull request merged**